src/cryptoadvance/specter/templates/includes/notifications.js

```diff
@@ -0,0 +1,377 @@
+{% include "includes/message_box.js" %}
+
+
+var userToken = null;
+
+/**
+ * creating a notification from JS
+ * 
+ * Example:
+ * createNotification('this is the title', {target_uis:['js_message_box', 'webapi'], body:'body line 1\nline 2', image:'/static/img/ghost_3d.png', timeout:3000})
+ */ 
```

Comment:
I think it would be useful to list all possible options here. Or is it listed somewhere else?

Reply:
extended the doc in  fc831bd5